tonic/models/vic/grid_params.py

```diff
@@ -637,34 +840,42 @@ def _run(args):
                         lai_src=args.lai_src,
                         fcan_src=args.fcan_src,
                         alb_src=args.alb_src,
-                        lake_profile=args.lake_profile)
+                        lake_profile=args.lake_profile,
+                        carbon=args.carbon)
```

Comment:
I should have caught this in your earlier PRs but we also need to add these arguments to the command line tool: https://github.com/jhamman/tonic/blob/master/scripts/vic_utils#L43-L85